API/src/routes/User/index.js

```diff
@@ -1,8 +1,27 @@
-const { ResponseHandler } = require(`../../utils`);
 const { UserService } = require(`../../microservices`);
+const { ResponseHandler } = require(`../../utils`);
 
-const BASE_URL = `/user`;
+const BASE_URL = `/assessment`;
```

Comment:
```suggestion
const BASE_URL = `/assessment`;
```
since this is in your user route, the url should reflect it. This should be /user and you should be sending the request to /user instead of /assessment on the frontend server.

Reply:
the sending thing you also need to correct is in your server/routes/UserAPI. You have /assessment where you should have /user

Reply:
Thank you.
I've applied those changes as well as replacing /assessment with /user in UserService